### PR TITLE
updated README with acceptance test directions

### PR DESCRIPTION
**Pull request by t-keazirian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,14 +37,31 @@ dotnet run --project TKeazirian.HTTPServer
...
 dotnet test
 ```
 
+### Acceptance Tests:
+To run the acceptance tests:
+1. You will need [Ruby][6] version `2.5.1`.  You can install Ruby using [brew][7] or the Ruby [docs][8]. 
+2. Once it is installed, run `ruby -v` to confirm you are using the appropriate version.
+3. Next, run `bundler install`.
+4. Start your HTTP server on port `5000`
+5. Run the acceptance tests using `rake test`.
```
[the remark this answers is not in the document]
Comment (Aoxian) [reply README.md:54]: I suggest explaining how to run your server on a specific port.  (looks like right now its a non-configurable code change - a nice update would be a way to pass a port override as a command line argument).

File: README.md [hunk 2]
```diff
@@ -37,14 +37,41 @@ dotnet run --project TKeazirian.HTTPServer
 
 This project uses xUnit.net for testing. Visit [this][5] website for more information and how to use xUnit.net.
 
+This project also uses Acceptance tests written in Cucumber Code found [here][9]. You can read more about Cucumber Code [here][10].
```
Comment (Aoxian) [inline README.md:40]: Why use "here" when you can just link directly?

```suggestion
This project also uses an [acceptance tests suite][9] written using [Cucumber][10]. 
```

File: README.md [hunk 3]
```diff
@@ -37,14 +37,41 @@ dotnet run --project TKeazirian.HTTPServer
...
+
 ### To Run the Tests:
 
 ```shell
 dotnet test
 ```
 
+### Acceptance Tests:
+**Setup**
```
Comment (Aoxian) [inline README.md:49]: When making a choice between [emphasis vs headers](https://cirosantilli.com/markdown-style-guide/#emphasis-vs-headers) in markdown, choose headers.

```suggestion
#### Setup
```

File: README.md [hunk 4]
```diff
@@ -37,14 +37,41 @@ dotnet run --project TKeazirian.HTTPServer
...
+
+1. You will need [Ruby][6] version `2.5.1`.  You can install Ruby using [brew][7] or the Ruby [docs][8]. 
+2. Once it is installed, run `ruby -v` to confirm you are using the appropriate version.
+3. Next, run `bundler install`.
+
+**Steps to Run**
```
Comment (Aoxian) [inline README.md:55]: When making a choice between [emphasis vs headers](https://cirosantilli.com/markdown-style-guide/#emphasis-vs-headers) in markdown, choose headers.

```suggestion
#### Steps to Run
```

**Comment by t-keazirian** [comment]
> A couple more clean-up comments and I think you are good to go.

Got it!  I do not know how to do [this](https://github.com/t-keazirian/HTTPServer/pull/11#discussion_r890319445) yet (implementation or how that would work) so can I hold off on giving these instructions until I know a little more how I could implement that?

**Comment by Aoxian** [comment]
> > A couple more clean-up comments and I think you are good to go.
> 
> Got it! I do not know how to do [this](https://github.com/t-keazirian/HTTPServer/pull/11#discussion_r890319445) yet (implementation or how that would work) so can I hold off on giving these instructions until I know a little more how I could implement that?

Here is a good example of a moment when you should make a decision instead of deferring. I understand the scenario. What are you going to do about it?

**Comment by t-keazirian** [comment]
> > > A couple more clean-up comments and I think you are good to go.
> > 
> > 
> > Got it! I do not know how to do [this](https://github.com/t-keazirian/HTTPServer/pull/11#discussion_r890319445) yet (implementation or how that would work) so can I hold off on giving these instructions until I know a little more how I could implement that?
> 
> Here is a good example of a moment when you should make a decision instead of deferring. I understand the scenario. What are you going to do about it?

I'll wait until I have a better understanding and merge this for now.  I added a ticket in the backlog so I don't forget.

**Comment by Aoxian** [comment]
> > > > A couple more clean-up comments and I think you are good to go.
> > > 
> > > 
> > > Got it! I do not know how to do [this](https://github.com/t-keazirian/HTTPServer/pull/11#discussion_r890319445) yet (implementation or how that would work) so can I hold off on giving these instructions until I know a little more how I could implement that?
> > 
> > 
> > Here is a good example of a moment when you should make a decision instead of deferring. I understand the scenario. What are you going to do about it?
> 
> I'll wait until I have a better understanding and merge this for now. I added a ticket in the backlog so I don't forget.

Great answer!  :shipit:

**Pull request merged**